src/components/gridList/useGridLayout.ts

```diff
@@ -42,18 +43,22 @@ const useGridLayout = (props: GridListBaseProps) => {
   }, [numberOfColumns, itemSpacing, keepItemSize ? undefined : _containerWidth]);
 
   const itemContainerStyle = useMemo(() => {
-    return {width: itemWidth /* + itemSpacing */, marginRight: itemSpacing, marginBottom: itemSpacing};
+    return {width: itemWidth /* , marginRight: itemSpacing, marginBottom: itemSpacing */};
```

Comment:
Why keep the comments?

Reply:
I usually keep comments in case we'll have a weird issue with these change it'll be easier to spot what was before
Usually after a comment "sits" for some time I delete it